src/main/resources/policies/document-rechten.rego

```diff
@@ -54,6 +54,7 @@ wijzigen {
     behandelaar.rol in user.rollen
     zaaktype_allowed
     document.zaak_open == true
+    document.definitief == false
```

Comment:
You checked this change with our PO right? Because right now all these policies are being validated by the municipalities.

Reply:
Where in https://dimpact.atlassian.net/browse/PZ-1356 does it say that this is only for the behandelaar role? I think the story needs to be updated right? Now it is not in sync with this code change.

Reply:
@RickWoltheus I still think we need to update the user story with what we just discussed right? but this PR is fine. :-)